compiler/one-cmds/how-to-use-one-commands.txt

```diff
@@ -149,6 +149,7 @@ one-optimize
 one-optimize provides network or operator transformation shown below.
 
 Current transformation options are
+- decompose_hardswish : This will decompose hardswish into multiple operators for special backends.
```

Comment:
in `circle2circle`, description is `Decompose HardSwish operator to Add, Mul and Relu6 operators`. Do you want to go with current one and fix also `circle2circle` or revise this to follow `circle2circle` ?

Reply:
I think it would be better to edit this description to the one on `circle2circle`.
I will update the commit. 
Thanks for pointing it out. 😺